rust/src/lib.rs

```diff
@@ -97,8 +97,6 @@ pub mod vacuum;
 pub mod checkpoints;
 #[cfg(all(feature = "arrow", feature = "parquet"))]
 pub mod delta_arrow;
-#[cfg(feature = "rust-dataframe-ext")]
```

Comment:
Could you also update the doc comment at the top of the module, which still mentions ADLS2?